### ISSUE TITLE
Accessibility and usability issues with Joplin

### ISSUE THREAD

**Issue by akash07k** [laurent22/joplin#7416]
## Description

There are several high impacting accessibility issues in Joplin due to which it can't be used with screen readers.
I'm a visually impaired developer and I'm dependent on screen readers for using my devices.
I was in need of a powerful and feature rich markdown based note taking app since a long time and was planning to develop one in C# .NET but coincidently stumbled upon Joplin.
It has all the features which I need and I feel that it would not be wise to develop such an app from scratch if the powerful app like Joplin already exists.
It is really a great app, but unfortunately, it has several accessibility issues due to which it can't be used with screen readers.
For example, the editor area is not accessible, whenever we type anything in the editor and try to read it via arrow keys, screen reader fails to read the content in most of the cases.
There are many unlabeled controls too.
Also, there are no headings/landmarks implemented for easily accessing the app's interface.
App's interface is challenging to navigate for screen reader users.
I want to request the developers to please implement the accessibility if it is possible.
I too am a dev and will be happy to help in implementation as well as testing of the accessibility implementation.
I was planning to send some PRs with the fixes but sadly the codebase is huge and I'm getting it very time consuming to become familiarized with the whole codebase.
Below are some links which can help in understanding more about accessibility:

[What is accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)
[Introduction to web accessiblity](https://www.w3.org/WAI/fundamentals/accessibility-intro/)
[Ultimate guide to accessibility](https://blog.hubspot.com/website/web-accessibility)

## Environment

Joplin version: 2.8.8.0
Platform: Windows
OS specifics: Windows 11

## Steps to reproduce
1. Open Joplin
2. Turn on a screen reader for example NVDA (free and open source)
3. Try to navigate in the apps interface and observe the control labels.
4. Try to type something in the editor and read it back with the arrow keys
5. Observe

## Describe what you expected to happen
Would be happy if the accessibility can be implemented natively, it will benefit everybody

## Logfile
Not applicable

**Comment by laurent22** [comment]
Hello, accessibility related tasks are tracked in this issue: https://github.com/laurent22/joplin/issues/4952

See also the accessibility label if you would like to help fix some of those issues: https://github.com/laurent22/joplin/issues?q=is%3Aissue+is%3Aopen+label%3Aaccessibility+

I will close this issue as a duplicate of #4952, but feel free to add any additional comments on the other issue. As you'll see unfortunately we haven't made much progress on this lately.

**Comment by akash07k** [comment]
Great, I'll give it a look
